wallet/src/de/schildbach/wallet/ExchangeRatesProvider.java

```diff
@@ -0,0 +1,568 @@
+/*
+ * Copyright 2011-2015 the original author or authors.
```

Comment:
why are we adding this? This is not part of the WallofCoins module. I do not believe we should add this, instead we should ask @tomasz-ludek where we should get the exchange rate data, or we need to abandon getting exchange rate data.

Reply:
This file was in the previous version of Dash Wallet from April 2017.  In our upgrades the exchange rates classes were moved and refactored to:  wallet/src/de/schildbach/wallet/data/ExchangeRatesProvider.java
wallet/src/de/schildbach/wallet/data/ExchangeRatesLoader.java
wallet/src/de/schildbach/wallet/data/ExchangeRate.java

Reply:
@rgenito All of this files are using in BuyDashFragment.Java.

Reply:
@rgenito  I have update BuyDashFragment.Java.